css/_components.team-photos.css

```diff
@@ -0,0 +1,166 @@
+/* Classes for team photos negative top margin BEGIN */
+.u-overflow-content {
+  @media (--small-ish) { overflow: visible; }
+}
+
+.u-mr-overflow-content {
+  @media (--small-ish) { margin-bottom: 8rem; }
+  @media (--medium-ish) { margin-bottom: 14rem; }
+}
+/* Classes for team photos negative top margin END */
+
+
+.c-team-photos {
+  margin-bottom: 2rem;
+  padding-left: 2rem;
+  padding-right: 2rem;
+
+  @media (--small-ish) {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 4rem;
+    margin-top: -9rem; /* Negative margin for overlapping with previous module */
+  }
+
+  @media (--medium-ish) {
+    margin-bottom: 6rem;
+    margin-top: -16rem;
+  }
+}
+
+.c-team-photos__item {
+  list-style: none;
+  position: relative;
+  text-align: center;
+
+  @media (--small-ish) { flex: 0 1 26.5%; }
+}
+
+.c-team-photos-link { display: block; }
```

Comment:
I believe this should be an element of the `c-team-photos` block. Same applies to all of the declarations below:
```css
.c-team-photos__link {}
.c-team-photos__img {}
.c-team-photos__caption {}
.c-team-photos__shapes {}
```
For `shapes` element you can keep already existing modifiers. ;)